pkg/provider/operators.go

```diff
@@ -57,6 +64,48 @@ func (op *Operator) String() string {
 	return fmt.Sprintf("csv: %s ns:%s subscription:%s targetNamespace=%s", op.Name, op.Namespace, op.SubscriptionName, op.TargetNamespace)
 }
 
+func (op *Operator) SetPreflightResults(allowInsecure bool) error {
+	bundleImage := op.InstallPlans[0].BundleImage
+	indexImage := op.InstallPlans[0].IndexImage
+	oc := clientsholder.GetClientsHolder()
+
+	// Create artifacts handler
+	artifactsWriter, err := artifacts.NewMapWriter()
+	if err != nil {
+		return err
+	}
+	ctx := artifacts.ContextWithWriter(context.Background(), artifactsWriter)
+
+	opts := []plibOperator.Option{}
+
+	// Check to make sure that the environment variable is set
+	if val := os.Getenv("PFLT_DOCKERCONFIG"); len(val) > 0 {
```

Comment:
See the other comment about environment variables.

Reply:
Done.  Moved to the envconfig parameter struct.